helm/aws-load-balancer-controller/templates/service-metrics.yaml

```
@@ -6,6 +6,7 @@ metadata:
   annotations:    
     giantswarm.io/monitoring: "true"
     giantswarm.io/monitoring-port: "{{ (split ":" .Values.metricsBindAddr)._1 | default 8080 }}"
+    giantswarm.io/monitoring-app-label: "aws-loadbalancer-controller"
```

```suggestion
    giantswarm.io/monitoring-app-label: "aws-load-balancer-controller"
```
to match the repo name, right?